R/read_gt3x.R

```diff
@@ -80,6 +81,34 @@ read_gt3x <- function(
     }
 
   }
+  if ("RAW" %in% names(log)) {
+    if ("Timestamp" %in% names(log$RAW)) {
+      check = anyDuplicated(log$RAW$Timestamp)
+      if (any(check)) {
+        warning(
+          "Duplicated timestamps in the data, this usually indicates an error",
+          call. = FALSE
+        )
+      }
+    }
+    xyz = c("X", "Y", "Z")
+    odd_value_threshold = 20
+    cn = paste0("Accelerometer_", xyz)
+    if (any(cn %in% names(log$RAW))) {
+      values = log[,intersect(cn, names(log$RAW))]
+      check = any(abs(values) > odd_value_threshold)
+      rm(values)
+      if (any(check)) {
+        warning(
+          paste0("Data values outside of ",
+                 odd_value_threshold,
+                 " threshold, ",
+                 "this usually indicates an error"),
+          call. = FALSE
+        )
+      }
+    }
```

Comment:
* This can also be cleaned up.
* Since `odd_value_threshold` is static, I don't see a need to assign it inside the function environment.
* If you're needing to test different values of `odd_value_threshold`, I recommend assigning it globally at the top of [read_gt3x_utils.R](https://github.com/paulhibbing/AGread/blob/master/R/read_gt3x_utils.R) as `.odd_value_threshold`. It'll be a bit of a hassle to change the value and then re-load the package, but I don't think this would be a suitable argument to add to the user-facing signature.

```suggestion
  if (any(.accel_names %in% names(log$RAW))) {
    
    aberrant_values <-
      names(log$RAW) %>%
      intersect(.accel_names) %>%
      log$RAW[ ,.] %>%
      abs(.) %>%
      {. > .odd_value_threshold}
        
    if (any(aberrant_values)) warning(
      "Detected values beyond ", odd_value_threshold, " g. ",
      "This usually indicates an error.", call. = FALSE
    )
      
    if (verbose) cat("  ............. COMPLETE")
        
  }
```

Reply:
Added